testdata/parts/badattach.raw

```diff
@@ -0,0 +1,6 @@
+Content-Type: ;
+	name=""
+Content-Transfer-Encoding: base64
+Content-Disposition: attachment;
+	filename=""
+
```

Comment:
This is actually the entirety of the MIME part that was received in the email the resulted in this PR.

Reply:
Done.